### ISSUE TITLE
Documentation for gradle kotlin dsl

### ISSUE THREAD

**Issue by corlaez** [JetBrains/spek#890]
I had troubles configuring with gradle kotlin dsl. I had to read a bunch of docs to solve it.

Ideally I would like to have the kotlin version in the docs.
Also specify that gradle 6.5 will also work (it is not entirely clear right now, in my opinion)

**Comment by corlaez** [comment]
This is the gradle kotlin dsl config:
```kt
tasks.test {
    useJUnitPlatform {
        includeEngines("spek2")
    }
}
```

**Comment by raniejade** [comment]
Thanks for the report! Documentation is indeed dated, I haven't touched it in a while now.

**Comment by raniejade** [comment]
Resolved by #891
